### DISCUSSION TITLE
php artisan make:filament-resource Customer --generate

### DISCUSSION THREAD

**Discussion by ihssane2022** [filamentphp/filament#13440] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

php artisan make:filament-resource Customer --generate   Error

Class "App\Models\Student" not found

PHP 8.2.12

**Comment by leandrocfe** [comment]
Probably, in somewhere you are using `Student` model without import this.
Check if you have this file: `app/Models/Student.php`
